### PR TITLE
Document that Qt 5.9 or later is required

### PR DESCRIPTION
**Pull request by lrineau**
## Summary of Changes

The code of CGAL_Qt5 and CGAL Qt5 demos requires Qt-5.9, but we kept documenting that the minimal version was 5.3. Now, document version 5.9 or later.

## Release Management

* Affected package(s): Installation
* License and copyright ownership: maintenance

### CODE REVIEW

File: Documentation/doc/Documentation/Installation.txt [hunk 1]
```diff
@@ -468,7 +468,7 @@ The exhaustive list of \sc{Qt}5 components used in demos is:
 `Core`, `Gui`, `Help`, `OpenGL`, `Script`, `ScriptTools`, `Svg`, `Widgets`,
 `qcollectiongenerator` (with `sqlite` driver plugin) and `Xml`.
 
-Having \sc{Qt}5 version 5.3.0 is recommended.
+The minimal version of \sc{Qt}5 is version 5.9.0.
```
Comment (lrineau) [inline Documentation/doc/Documentation/Installation.txt]: @CGAL/geometryfactory Is that a correct wording?

Comment (janetournois) [reply Documentation/doc/Documentation/Installation.txt]: '\sc{Qt}5.9.0 or later is required' ?

**Comment by lrineau** [comment]
I have modified the wording:
> Qt version 5.9.0 or later is required.

Once compiled with Doxygen, that displays:
> Screenshot_20190430_165150

**Comment by maxGimeno** [comment]
Successfully testsed in [Ic-29](https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-29.shtml)

**Pull request merged**